modules/_node-scripts/util/writeTimings.mjs

```diff
@@ -0,0 +1,39 @@
+import {constants} from 'fs';
+import * as fs from 'fs/promises';
+import * as path from 'path';
+
+import getRootDir from './getRootDir.mjs';
+
+export default async function writeTimings(start, endConfig) {
```

Comment:
Just a random thought for the future... It'd be interesting if we could use this  for some sort of data gathering in CI and could POST the results to some endpoint and that way we could track it overtime

Reply:
Yes. In fact I proposed to track build times to the CI team in the context of a different request. So we may want to address this feature in the future as a joint effort :+1: .